samples/sample_plugins/rotate_opencl/src/opencl_filter_dx11.cpp

```diff
@@ -36,37 +32,25 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #define DX_MEDIA_SHARING
 #define CL_DX9_MEDIA_SHARING_INTEL_EXT
 
-#include <CL/opencl.h>
+#include <CL/cl_ext.h>
 #include <CL/cl_d3d11.h>
 
 using std::endl;
 
-#define INIT_CL_EXT_FUNC(x)    x = (x ## _fn)clGetExtensionFunctionAddress(#x);
 #define SAFE_OCL_FREE(P, FREE_FUNC)  { if (P) { FREE_FUNC(P); P = NULL; } }
 
 #define MAX_PLATFORMS       32
```

Comment:
please, check whether we need these 2 defines? I thought we've moved to std::vector?